### ISSUE TITLE
Custom Locations

### ISSUE THREAD

**Issue by FreshImmuc** [jlesage/docker-nginx-proxy-manager#161]
Hey,

Custom locations don't work for me, or im just doing it wrong. Everything else works fine. I can also acess the IP:PORT/PATH

**GUI configuration:** 

![image](https://user-images.githubusercontent.com/65733898/110226072-ec314b80-7eeb-11eb-8413-ad1e05149142.png)

![image](https://user-images.githubusercontent.com/65733898/110226033-8e046880-7eeb-11eb-97ac-c937f38d4586.png)

**FILE configuration / path:**

![image](https://user-images.githubusercontent.com/65733898/110226050-d3289a80-7eeb-11eb-85e3-3aab62755d27.png)

Port 8080 goes to apache webserver.

You can try go to https://fireplanet.net, then you will get to the homepage and everything works fine  

if you go to https://fireplanet.net/404 (the path from the screenshot) everything works fine too.
but if you go to https://404.fireplanet.net then the normal 404 Page shows up, but not the index.html file, that is in the path

**Comment by jlesage** [comment]
Not sure if I understand the problem.  What are you expecting to happen when accessing https://404.fireplanet.net ?

**Comment by FreshImmuc** [comment]
The norml site shows up. The same, if you go to https://fireplanet.net But I configured a custom location. A custon location is that you can configure another folder where the html files are stored right?

**Comment by jlesage** [comment]
No, custom location is used to forward to a sub-path to a different host.  For example, you can have:
 * http://www.example.com that forwards to host A.
 * http://www.example.com/custom that forwards to host B.

**Comment by FreshImmuc** [comment]
Okay, but is there a way to set a folder as the Apache "html" folder for a Subdomain?

**Comment by jlesage** [comment]
Maybe you can try to add a custom location, where:
  - Location is `/`.
  - Forward IP is `<ip of your webserver>/404`.

**Comment by FreshImmuc** [comment]
Okay. So this "worked" but heres another Problem: If i go to  "path" like https://fireplanet.net/404, It redirects me to https://fireplanet.net:8080/404 and with the Port, the website doesn't work... If you remove the port, everything works again
